stage1_fly/enter/main.go

```diff
@@ -15,15 +15,76 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"io/ioutil"
 	"os"
+	"syscall"
 
 	rktlog "github.com/coreos/rkt/pkg/log"
 )
 
-var log *rktlog.Logger
+var (
+	debug   bool
+	podPid  string
+	appName string
+
+	log  *rktlog.Logger
+	diag *rktlog.Logger
+)
+
+func init() {
+	flag.BoolVar(&debug, "debug", false, "Run in debug mode")
+	flag.StringVar(&podPid, "pid", "", "Pod PID")
+	flag.StringVar(&appName, "appname", "", "Application (Ignored in rkt fly)")
+
+	log, diag, _ = rktlog.NewLogSet("fly-enter", false)
+}
+
+func getRootDir(pid string) (string, error) {
+	rootLink := fmt.Sprintf("/proc/%s/root", pid)
+
+	return os.Readlink(rootLink)
+}
+
+func execArgs() error {
+	argv0 := flag.Arg(0)
+	argv := flag.Args()
+	envv := []string{}
```

Comment:
Note to self: fly should write env var in a stage1 file (under `/rkt/env/`) and here we should set them back and ensuring that there is a sane `$TERM` set.